.github/CODEOWNERS

```diff
@@ -466,7 +465,7 @@
 /pkg/fleet/                             @DataDog/fleet @DataDog/windows-agent
 /pkg/fleet/installer/setup/djm/         @DataDog/fleet @DataDog/data-jobs-monitoring
 /pkg/pidfile/                           @DataDog/agent-runtimes
-/pkg/persistentcache/                   @DataDog/agent-metric-pipelines
+/pkg/persistentcache/                   @DataDog/agent-runtimes
```

Comment:
@pgimalac More of a curiosity - how does this relate to the ownership transfer (I'm not overly familiar with how this pkg is used)

Reply:
I didn't know about it either, I checked and it's only used by checks (some Go + Python), rtloader exposes an API for Python checks to use this package, so I figured we should get the ownership along with rtloader / the collector